### DISCUSSION TITLE
branching

### DISCUSSION THREAD

**Discussion by cauerego** [kcubeterm/achoz#13] [General]
i just pushed try commits to the main branch.

this is highly not recommended, the way i did it.

but for such an early alpha stage and small base, it's obviously fine.

we may soon need to get a branching flow to keep using git, nevertheless.

i hope we can migrate to fossil before that... in which i'll have to learn all over again (not that i ever branched that much...) 😁

**Comment by kcubeterm** [comment]
Yea, Push changes to different branch first, once you finished with your features, simply merge it. Since you are maintaining branch and pushes changes on github as well, other members could review it easilly. Same like i do with server branch seperately
